src/Connect.js

```diff
@@ -43,8 +43,15 @@ class Connect {
     this.mobileTransport = opts.mobileTransport || transport.url.send()
     this.onloadResponse = transport.url.getResponse()
```

Comment:
@zachferland it should be possible to pass in custom function here:
```
this.onloadResponse = opts.onloadResponse || transport.url.getResponse()
```

Reply:
yeah, actually not function, it is just a response string set to this.onloadResponse, but yeah could allow people to pass in an initial response string on instantiation, makes sense